doc/api/cv.md

```diff
@@ -1,5 +1,9 @@
-Cyclic Voltamograms
-===================
+---
+github_url: https://github.com/echemdb/svgdigitizer/blob/master/svgdigitizer/cv.py
+---
+
+Cyclic Voltammograms
```

Comment:
Hopefully we've now found the last voltamogram.

Reply:
I quickly replaced two other instances. Hope dies last. :)